build/configs/artik053/artik053_download.sh

```diff
@@ -73,6 +98,12 @@ main()
 				flash_write wlanfw ../../bin/wlanfw.bin;	\
 				flash_write os ../../../../output/bin/tinyara_head.bin;	\
 			exit'
+			if [ "${CONFIG_FS_ROMFS}" == "y" ]; then
+				${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c '
+					flash_write rom ../../../../output/bin/romfs.img;	\
```

Comment:
Please consider calling the openocd command just once -- this can be made with constructin the parameters list before actually running the openocd command:

> OPENOCD_PARAMETERS=...
>if [ "x${CONFIG_FS_ROMFS}" == xy ]; then
>  OPENOCD_PARAMETERS+=" flash_write rom <path_to_romfs_image>"
>fi
>(...)
> [ -z "${OPENOCD_PARAMETERS}" ] || ${OPENOCD} -f artik053.cfg -c "${OPENOCD_PARAMETERS}"

This would open a path to further parameterize download script to flash image parts independently.